src/core/formatter.mli

```diff
@@ -0,0 +1,12 @@
+(* 
+ * Formatter interface 
+ * DURAND - MARAIS © 2019
+ *)
+
+open Monad
+
+(* Recover string corresponding to the mustache template *)
```

Comment:
(\*\* needed

Reply:
I don't understand the comment.

Reply:
Need a double \* instead of one to be accept by odoc